webapp/packages/plugin-connections/src/PublicConnectionForm/PublicConnectionFormService.ts

```diff
@@ -60,7 +60,7 @@ export class PublicConnectionFormService {
       executorHandlerFilter(
         () => !!this.formState && this.optionsPanelService.isOpen(formGetter),
         async (event, context) => {
-          if (event === 'before' && this.userInfoResource.data === null) {
+          if (event === 'before' && !this.userInfoResource.isData()) {
```

Comment:
for this place it's better to use `this.userInfoResource.isAnonymous()` because we want to display this message only if user is anonymous and this action can lead to login as a user